app/src/main/java/com/segment/analytics/android/integrations/appsflyer/AppsflyerIntegration.java

```diff
@@ -117,6 +125,50 @@ public void track(TrackPayload track) {
         logger.verbose("appsflyer.trackEvent(null, %s, %s)",context, event, properties);
     }
 
+    static class ConversionListener implements AppsFlyerConversionListener {
+        final Analytics analytics;
 
-}
+        public ConversionListener(Analytics analytics) {
+            this.analytics = analytics;
+        }
+
+        @Override public void onInstallConversionDataLoaded(Map<String, String> conversionData) {
+            trackInstallAttributed(attributionData);
+        }
+
+        @Override public void onInstallConversionFailure(String errorMessage) {
+
```

Comment:
I couldn't tell if your SDK does logging here or not. If it does not, it might be worthwhile to add logging calls in this callback.